src/mmw/js/src/analyze/templates/pointSourceTable.html

```diff
@@ -37,10 +37,23 @@
   </table>
 </div>
 <div class="row paging-ctl-row">
-  <button class="btn-prev-page btn btn-primary">
-    <span aria-hidden="true">&larr;</span>
-  </button>
-  <button class="btn-next-page btn btn-primary">
-    <span aria-hidden="true">&rarr;</span>
-  </button>
+    <p>
+        {% if totalPages %}
+            {{currentPage}} of {{totalPages}}
+        {% endif %}
+    </p>
+    {% if hasPreviousPage %}
+        <button class="btn-prev-page btn btn-primary"
+          data-toggle="tooltip" title="Previous Page"
+          data-placement="right">
+            <span aria-hidden="true">&larr;</span>
+        </button>
+    {% endif %}
+    {% if hasNextPage %}
```

Comment:
I'd prefer the button to stay, but be disabled, however our styling for the `disabled` attribute isn't great:

![screenshot from 2016-11-01 17 48 34](https://cloud.githubusercontent.com/assets/1014341/19908855/6c30e10c-a05b-11e6-9552-5df5dd4cfd1b.png)

I think this will work as is, though.